### ISSUE TITLE
Ability to see the hostname

### ISSUE THREAD

**Issue by LoZio** [cydrobolt/polr#396]
In a multi-server configuration it should be useful to see the server name

## Expected Behavior
Have a config variable to enable this at two levels:
APP_SHOW_SERVER_NAME = 1
Show the server name in the top bar of the app
The same set to 2 can display the server name ALSO as a comment inside the html of the login page
Set to 0: non server name (same as today)

## Current Behavior
Cannot know which server I'm going to

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version or latest commit hash (`git rev-parse HEAD`):
* Environment name and version (e.g. Chrome 39, PHP 7.0, etc):
* Instance link (optional):

**Comment by sndrr** [comment]
How is your setup? One database with multiple frontend webservers (HA-setup)?

**Comment by LoZio** [comment]
Confirm

**Comment by LoZio** [comment]
AWS ELB in front to a pool of servers. Session stickyness to be able to configure the application. This is sub-optimal for load distribution but required since sessions are not kept in the DB so switching from one server to another breaks things. Would be useful to put the session in the DB/Redis to avoid using sticky connections. Not a real problem when you have a lot of clients clicking since they are evenly spread in any case.

**Comment by overint** [comment]
@LoZio 

You can set the session driver to redis in the `.env` file, line `106`,
```
...
CACHE_DRIVER=file
SESSION_DRIVER=redis
QUEUE_DRIVER=database
...
```

**Comment by overint** [comment]
Would showing the server name on `/about-polr` be an adequate solution? I don't think we need to be showing it in the nav.

**Comment by LoZio** [comment]
Yes of course, but having it handy (even in some fine print format) whould be better.
Looking at my use-case also adding a specific X-HEADER in the redirect reply would be useful.

**Comment by LoZio** [comment]
@overint Where do I find doc about configuring redis settings?

**Comment by overint** [comment]
https://lumen.laravel.com/docs/5.1/session

**Comment by LoZio** [comment]
Ok, this shows me how to code using laravel and redis. My question was: is there some PolR documentation about how to setup redis sessions? You mentioned to change a line adding
`SESSION_DRIVER=redis`
but where do I find PolR docs about where to set the server name, port, cluster (if any), db number, user and password and so on?
Or where you suggesting: this is how to add the support to PolR go code it yourself?

**Comment by overint** [comment]
You will need to add the redis vars to your .env, as below:

```
REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

```

**Comment by LoZio** [comment]
Sorry for the late reply, had time to test only today.
Just adding SESSION_DRIVER=redis (and filling all the redis variables) results in "Whoops, looks like something went wrong.".
Redis is online (on another host) and working.
Where do I find some log to further investigate? Apache error log contains no info, access log says it was a 500.

**Comment by LoZio** [comment]
Also tried to create a config/database.php file from some online examples with no luck.

**Comment by overint** [comment]
You will likely need to check your `error.log` not your `access.log`

**Comment by LoZio** [comment]
No such file, but found `lumen.log` that says
`[2018-05-04 08:27:43] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Fatal error: Class 'Illuminate\Redis\RedisServiceProvider' not found in /mnt/web/polr/vendor/laravel/lumen-framework/src/Application.php:268`
